lib/KLogo/__tests__/KLogo.spec.js

```diff
@@ -0,0 +1,62 @@
+import { renderComponentForVisualTest, takeSnapshot } from '../../../jest.conf/visual.testUtils';
+
+describe.visual('KLogo', () => {
+  const snapshotOptions = { widths: [400], minHeight: 512 };
+  describe('Render KLogo with different props', () => {
+    it('Render a default logo', async () => {
+      await renderComponentForVisualTest('KLogo', { altText: 'Kolibri default logo', size: '150' });
+      await takeSnapshot('KLogo - default', snapshotOptions);
+    });
+    it('Render a logo with background', async () => {
+      await renderComponentForVisualTest('KLogo', {
+        altText: 'Kolibri Logo with background',
+        size: '150',
```

Comment:
Setting `size` is not necessary in this test. Let's remove it, so the test focuses only on the background - that's its main purpose. And can you apply this to all other tests where this comment is relevant?